cmd/history.go

```diff
@@ -99,15 +94,10 @@ func displayUpdate(updates []backend.UpdateInfo, opts display.Options) {
 		timeCreated := humanize.Time(timeStart)
 		timeEnd := time.Unix(update.EndTime, 0)
 		duration := timeEnd.Sub(timeStart)
-
-		space := 2
-		fmt.Printf("%*sUpdated %s took %s\n", space, "", timeCreated, duration)
+		fmt.Printf("%2sUpdated %s took %s\n", "", timeCreated, duration)
 
 		empty := func(s string) bool {
-			if len(strings.TrimSpace(s)) == 0 {
-				return true
-			}
-			return false
+			return len(strings.TrimSpace(s)) == 0
```

Comment:
Nit: What do you think of the name `isEmpty` instead to make the code more readable?